vignettes/Probability-aggregation.Rmd

```diff
@@ -17,25 +17,25 @@ knitr::opts_chunk$set(
 
-In this vignette, we will explore this challenge, look at the theory behind it and explore possible solution. The Great Reed Warbler (18IC), we serve us as example to illustrate the solutions. 
+In this vignette, we will explore this challenge, look at the theory behind it and explore possible solution using Great Reed Warbler (18IC) as an example. 
 
 ## Problem presentation
 
 For each stationary period, we estimate the probability of the position $\textbf{x}$ of the bird based on a pressure timeseries measured by the geolocator $P_{gl}[t] \quad \forall t \in [1,\ldots,n]$, which in a Bayesian framework can be written as, 
 $$p(\textbf{x}\mid P_{gl}[1],\ldots,P_{gl}[n]) \propto p(P_{gl}[1],\ldots,P_{gl}[n] \mid \textbf{x}) p(\textbf{x}).$$
 
-We are interested here in determining the likelihood term $p(P_{gl} \mid \textbf{x})$ using quantify the probability of observing the timeserie $P_{gl}$ knowing that the bird is at location $\textbf{x}$. To quantify this probability, we look at the mismatch between $P_{gl}$ and the pressure timesrie of the ERA5 database $P_{ERA5}(\textbf{x})$ extracted at location $\textbf{x}$. We can reformulate the likelihood as as a probability function $f$ of an error term $\varepsilon(\textbf{x})$ which measure a distance between $P_{gl}$ and $P_{ERA5}(\textbf{x})$
+We are interested here in determining the likelihood term $p(P_{gl} \mid \textbf{x})$ which represents the probability of observing the timeseries $P_{gl}$ knowing that the bird is at location $\textbf{x}$. To quantify this probability, we look at the mismatch between $P_{gl}$ and the pressure timeseries of the ERA5 database $P_{ERA5}(\textbf{x})$ extracted at location $\textbf{x}$. We can reformulate the likelihood as as a probability function $f$ of an error term $\varepsilon(\textbf{x})$ which measures a distance between $P_{gl}$ and $P_{ERA5}(\textbf{x})$
 $$p(P_{gl} \mid \textbf{x}) = f(\varepsilon[1],\ldots,\varepsilon[n])$$
 
-This formulation of the problem help us to split our problem in two: first defining an error term $\varepsilon[t]$ and secondly defining the likelihood function $f$.
+This formulation of the problem helps us to split our problem in two: first defining an error term $\varepsilon[t]$ and secondly defining the likelihood function $f$.
 
 ## Error term
-In most cases, we would expect to measure the error term with a simple difference $P_{ERA5}(\textbf{x})[t]-P_{gl}[t].$ However, within a ERA5 grid cell of 9-30km, we can expect a wide range of altitude at which the bird can be location. As such there any offset between the two timeserie can be explained by the difference of altitude.
+In most cases, we would expect to measure the error term with a simple difference $P_{ERA5}(\textbf{x})[t]-P_{gl}[t].$ However, within a ERA5 grid cell of 9-30km, we can expect a wide range of altitude at which the bird can be located. As such any offset between the two timeseries might be due to nothing more than a difference of altitude.
 
```

Comment:
The idea of de-meaning the data only makes sense if we are already assuming that the residuals are Gaussian with mean zero. If the residuals are not assumed Gaussian, then de-meaning won't maximize the likelihood.  You *do* assume the residuals are zero-mean Gaussian, so this is all fine, but you don't introduce the assumption until later, so this part makes the reader a bit suspicious until we get a bit further.